packages/blockchain-wallet-v4-frontend/src/modals/ExchangeDetails/template.success.js

```diff
@@ -65,7 +65,7 @@ const ExchangeDetails = props => {
         {status === 'complete' &&
           <Notice>
             <Text size='13px' weight={300}>
-              <FormattedMessage id='modals.exchangedetails.explain' defaultMessage='Your exchange is complete.' />
+              <FormattedMessage id='modals.exchangedetails.explain' defaultMessage='Your exchange is complete. ' />
```

Comment:
instead of adding a space to the message, i'd suggest placing a `span` between the two formatted messages instead.
`<span>&nbsp;</span>`